### ISSUE TITLE
Prefix hidden to screen readers in inputs

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2082]
## Description

The prefix of the inputs is currently disconnected to input label or description, this can cause context problems for screen reader users, the best solution is to hide the prefix and give that information to the user using the label or the description of the field.

This recommendation of use should be included in BF to guide designers in order to provide an accesible experience when using prefixes in fields.

## Tasks

- [x] [Update specs](https://www.figma.com/design/e8d3PkXitUJ1Jwgc9Y3mNc/branch/yzmETXq2WosXeg1Izme8L2/%F0%9F%94%B8-Input-Fields-%26-Forms-Component-Specs?m=auto&node-id=3256-3982&t=JWrFjgqOCu3bqew1-1)
- [x] Design documentation
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-15842)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-11108)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2160)
